packages/react-composites/tests/browser/chat/ErrorBar.test.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { test } from './fixture';
+import { loadPage, stubMessageTimestamps, waitForCompositeToLoad } from '../utils';
+import { expect } from '@playwright/test';
+
```

Comment:
Should we add a test to verify that there's no ErrorBar displayed if there's no error?

Reply:
+1, thanks for calling that out.

I will add one to the following PR that introduces the remaining tests. This one mostly uses one test to get the boilerplate down.